outlines/routing.md

```diff
@@ -0,0 +1,40 @@
+# Routing
+
+* What is routing, and how it's different in a client-rendered app
+    * Uses for the URL
+        * Bookmarking
+        * Sharing
+        * Back/forward
+    * It's a useful serialization of some share-able, bookmarkable client-side state
+        * Not everything needs to be a route - consider that you can just store JavaScript variables, or use local storage for temporary information
+    * Routing is not related to data loading or authorization in a client-side app. For those, go see the security and data loading chapters
+* Flow Router intro
+* Creating routes
+    * How to define a basic route
+    * How to accept route parameters and URL pattern matching
+    * When to use query parameters vs. path parameters
+* Getting information about the current route
+    * Getting the parameters
+    * Highlighting the currently active route
+* Using the router to display templates/pages
+    * Layouts
+    * Using Blaze Layout
+    * Most of the logic is inside a template which represents a page. Read more in the data loading and Blaze chapters
```

Comment:
Using Layouts for business logic (i.e. authorization). Link to Blaze chapter? 

I guess what I am saying here is that we need to say what the pattern is for "deny access unless logged in". People will look here in the routing article. If the answer is "use layouts", I think we have to say more because the only other place we mention them is in the Blaze chapter, which React/Angular people won't read.